index.js

```diff
@@ -45,3 +45,84 @@ snackBtnClose.forEach((closeBtn) =>
 );
 
 hideAllSnack();
+
+/* For Rating */
+
+const ratingContainers = document.querySelectorAll(".rating-container");
+const stars = document.querySelectorAll(".star");
+
+const inActiveStarClasses = "fa fa-star-o star";
+const activeStarClasses = "fa fa-star star star-active";
+const hoverStarClasses = "fa fa-star star star-hover";
+
+function findStarsIndex(starElement) {
```

Comment:
You can try making use of ES6 arrow functions to make your functions as that will help you get into the habit as well as allow you to practise ES6 syntax for functions.

Reply:
I generally use arrow functions if the function block is small or when I need the return values. That was my thought process behind this.